packages/calcite-components/src/components/text-area/text-area.stories.ts

```diff
@@ -85,34 +85,41 @@ export const insideContainerWithHeightAndWidth_TestOnly = (): string =>
 
 export const validationMessageAllScales_TestOnly = (): string => html`
   <style>
-    .container {
+    .wrapper {
       display: flex;
-      flex-direction: column;
-      width: 420px;
-      height: 80px;
-      gap: 45px;
+      width: 800px;
+      height: 800px;
+      max-width: 100%;
+      padding: 64px;
+      gap: 10px;
     }
   </style>
-  <div class="container">
-    <calcite-text-area
-      scale="s"
-      status="invalid"
-      validation-message="This field is required."
-      validation-icon="frown"
-    ></calcite-text-area>
-    <calcite-text-area
-      scale="m"
-      status="invalid"
-      validation-message="Less than the minimum length of 6 characters"
-      validation-icon
-      value="Hi"
-    ></calcite-text-area>
-    <calcite-text-area
-      scale="l"
-      status="invalid"
-      validation-message="Exceeds the maximum length of 9 characters"
-      validation-icon
-      value="Lorem ipsum"
-    ></calcite-text-area>
+  <div class="wrapper">
```

Comment:
Do you need a wrapper per component or does the parent one suffice? If you need one per component, can you add a class each component wrapper vs duplicating the inline styles?